.github/workflows/build-and-test.yml

```diff
@@ -63,20 +63,19 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Make sure Postgres is ready and init
+        env:
+          PGPASSWORD: postgres
+        run: |
+          psql -h localhost -U postgres -f ./common/postgres-init.sql
+
       - name: Set up JDK
         uses: actions/setup-java@v2
         with:
           java-version: '17'
           distribution: 'temurin'
           cache: 'gradle'
 
-      - name: Make sure Postgres is ready and init
-        env:
-          PGPASSWORD: postgres
-        run: |
-          pg_isready -h localhost -t 10
```

Comment:
I see the difference between this removed block and the added block on L66-71 is this line. Was it intentional to remove `pg_isready` check and move the block one step up?

Reply:
Sorry I don't remember fully, as this was a change made at the beginning of this journey, but I know that the replacement of this code block was wo match more similarly what was in wds for my understanding. I think the `pg_isready` got lost in the process, and nothing broke without it, but my next commit will add it back.